### ISSUE TITLE
Default value isn't set in Interface despite being set in Entity

### ISSUE THREAD

**Issue by loevgaard** [hostnet/entity-plugin-lib#49]
I have the following entity:

```php
<?php
namespace Loevgaard\DandomainFoundation\Entity;

use Loevgaard\DandomainFoundation\Entity\Generated\OrderInterface;
use Loevgaard\DandomainFoundation\Entity\Generated\OrderTraits;

/**
 * @ORM\Entity()
 */
class Order implements OrderInterface
{
    use OrderTraits;

    public function populateFromApiResponse($data, bool $populateEmbedded = false) : OrderInterface
    {
        // ...
    }
}
```

This results in this generated OrderInterface:

```php
<?php
namespace Loevgaard\DandomainFoundation\Entity\Generated;

interface OrderInterface
{
    public function populateFromApiResponse($data, \boolean $populateEmbedded = );
}
```

As you can see the default value for $populateEmbedded isn't set in the interface, and results in a syntax error.

**Comment by nicoschoenmaker** [comment]
We had this error as well when adding typehints to our code. As far as I know this was fixed it in `2.1.2`.

See this test case
https://github.com/hostnet/entity-plugin-lib/blob/master/test/Fixtures/DefaultValueParams.php#L49-L54

Resulting in this interface
https://github.com/hostnet/entity-plugin-lib/blob/master/test/Fixtures/DefaultValueParamsInterface.php#L41-L44

Which version are you using?

**Comment by loevgaard** [comment]
Okay, I think I may have looked too much at the readme which says to use version 1. I should have noticed there was a version 2. Everything seems fine now :-)

Thank you!

**Comment by nicoschoenmaker** [comment]
Ai, sorry about that. But cool that updating fixes the problem! Opened #50 so that the next person won't get misdirected :)
